model/k8s/registry.yaml

```diff
@@ -687,3 +687,74 @@ groups:
           [Kubernetes Resource Quotas documentation](https://kubernetes.io/docs/concepts/policy/resource-quotas/#object-count-quota)
           for more details.
         examples: [ 'count/replicationcontrollers' ]
+      - id: k8s.pod.status.reason
+        type:
+          members:
+            - id: evicted
+              value: 'Evicted'
+              brief: 'The pod is evicted.'
+              stability: experimental
+            - id: node_affinity
+              value: 'NodeAffinity'
+              brief: 'The pod is in a status because of its node affinity'
+              stability: experimental
+            - id: node_lost
+              value: 'NodeLost'
+              brief: >
+                The reason on a pod when its state cannot be confirmed as kubelet is unresponsive
+                on the node it is (was) running.
+              stability: experimental
+            - id: shutdown
+              value: 'Shutdown'
+              brief: 'The pod is shutdown'
+              stability: experimental
+            - id: unexpected_admission_error
+              value: 'UnexpectedAdmissionError'
+              brief: >
+                The pod was rejected admission to the node because of an error during admission
+                that could not be categorized.
+              stability: experimental
+        stability: experimental
+        brief: >
+          The reason for the pod state. Corresponds to the `reason` field of the:
+          [K8s PodStatus](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.33/#podstatus-v1-core)
+        examples: ['Evicted', 'NodeAffinity']
+      - id: k8s.pod.phase
+        type:
+          members:
+            - id: pod_pending
+              value: 'PodPending'
```

Comment:
Why are these prefixed with Pod?  Should this just be pending/running/succeeded?